lib/widgets/task_widget.dart

```diff
@@ -23,7 +23,7 @@ class TaskWidget extends StatefulWidget {
 class _TaskWidgetState extends State<TaskWidget> {
   Future<CameraDescription> _getCamera() async {
     final cameras = await availableCameras();
-    return cameras.first;
+    return cameras[1];
```

Comment:
What if this camera doesn't exist? there should at least be a check for null. Also, explain somewhere why only front camera can be used, otherwise users will be confused. And if there is no front camera, then use first camera and set luxValue invalid (-1 or null)